sentinel/src/transitions/accept_patient_reports.js

```diff
@@ -175,10 +173,27 @@ const findValidRegistration = (doc, config, registrations) => {
 const addReportUUIDToRegistration = (doc, config, registrations, callback) => {
     const validRegistration = registrations.length && findValidRegistration(doc, config, registrations);
     if (validRegistration) {
-      return db.medic.put(validRegistration, callback);
+      return db.medic.put(validRegistration, (err) => {
+              if (err) {
+                if (err.name === 'conflict') {
+                  db.get(validRegistration._id, function(err, doc) {
+                    if (err) { 
+                      callback(err);
+                    }
+                    
+                    validRegistration._rev = doc._rev;
+                    db.medic.put(validRegistration, callback);
+                  });
+                } else {
+                  callback(err);
```

Comment:
you need to return here, otherwise the next `callback(null, true);` will also be called.